react/features/toolbox/components/native/Toolbox.js

```diff
@@ -46,37 +46,10 @@ type Props = {
     dispatch: Function
 };
 
-/**
- * The type of {@link Toolbox}'s React {@code Component} state.
- */
-type State = {
-
-    /**
-     * The detected width for this component.
-     */
-    width: number
-};
-
 /**
  * Implements the conference toolbox on React Native.
  */
-class Toolbox extends Component<Props, State> {
-    state = {
-        width: 0
-    };
-
-    /**
-     *  Initializes a new {@code Toolbox} instance.
-     *
-     * @inheritdoc
-     */
-    constructor(props: Props) {
-        super(props);
-
-        // Bind event handlers so they are only bound once per instance.
-        this._onLayout = this._onLayout.bind(this);
-    }
-
+class Toolbox extends Component<Props> {
```

Comment:
Nice cleanup! PureComponent, since we are here?

Reply:
I checked that, but we need that to look good. Removed it and I didn't like what I saw :)

Reply:
sure!